application/frontend/controllers/UserController.php

```diff
@@ -8,11 +8,16 @@
 
 class UserController extends BaseRestController
 {
-    public function actionIndex()
+    public function actionIndex() // GET /user
     {
         return User::find()->all();
     }
 
+    public function actionView($employeeId) // GET /user/abc123
+    {
+        return User::findOne(['employee_id' => $employeeId]);
```

Comment:
You mentioned that you might need to add more logic in order to return a `204` when no such user is found ([spec](https://github.com/silinternational/idp-id-broker/blob/develop/api.raml#L323)).